src/encoding.md

```diff
@@ -278,6 +279,50 @@ fn run() -> Result<()> {
 quick_main!(run);
 ```
 
+[ex-base64]: #ex-base64
+<a name="ex-base64"></a>
+## Encode and decode base64
+
+[![base64-badge]][base64] [![cat-encoding-badge]][cat-encoding]
+
+The [`base64`] crate provides [`encode`] and [`decode`] functions.
```

Comment:
I would suggest to to remove the first sentence and add links to [`encode`] and [`decode`] in the second sentence here.